providers/summary/scancode.js

```diff
@@ -28,22 +28,30 @@ class ScanCodeSummarizer {
     const result = {}
     this.addDescribedInfo(result, coordinates, harvested)
     const buckets = this.computeFileBuckets(harvested.content.files, facets)
-    set(result, 'licensed.facets', {})
-    const facetsObject = get(result, 'licensed.facets')
-    const declaredLicenses = new Set()
+    let declaredLicenses = new Set()
+    let packageLicenses = new Set()
+    for (const fileIdx in harvested.content.files) {
+      const file = harvested.content.files[fileIdx]
+      if (!file.path.toLowerCase().includes('/')) {
```

Comment:
Only look at files that are in the root folder. File paths in the root folder will not have a slash in them

Reply:
OK. we should not really try to fight whole communities like that. So we need to adapt. there will be other cases as well I'm sure. How do you propose to accommodate the npm scenario?

Reply:
I decided to look at the last component of each file path instead. This allows us to find all the license/package files even though they are not in their proper location in the root folder. Its tough to expect everyone to follow the open source guidelines because it seems like most everyone is doing it wrong right now anyway. I do think we should take into account the fact that the license/package files are not in the root folder in the clearlyDefined score.